host/sgx/sgxsign.c

```diff
@@ -572,6 +572,8 @@ static oe_result_t _init_sigstruct(
     uint64_t attributes,
     uint16_t product_id,
     uint16_t security_version,
+    const uint8_t* family_id,
+    const uint8_t* ext_product_id,
```

Comment:
OE coding style is to avoid abbreviations.
```suggestion
    const uint8_t* extended_product_id,
```

Reply:
Updated to align with OE coding style

Reply:
Thanks. Lost change when I updated the PR. Updated.